submissions/ArthurGorbenko/Memory Pair Game/js/script.js

```diff
@@ -0,0 +1,128 @@
+const valuesOfBackPos = [
+  "20px 0px",
+  "20px -180px",
+  "20px -350px",
+  "20px -525px",
+  "-190px 0px",
+  "-390px 0px",
+  "-600px 0px",
+  "-810px 0px"
+];
+
+let cards = document.getElementsByClassName("card");
+const container = document.querySelector(".container");
+
+for (let i = 0; i < cards.length; i++) {
+  addElements(cards[i], i);
+}
+
+function addElements(element, i) {
+  let flipper = document.createElement("div");
+  flipper.className = "flipper";
+  element.appendChild(flipper);
+
+  let front = document.createElement("div");
+  front.className = "front";
+  flipper.appendChild(front);
+
+  let back = document.createElement("div");
+  back.className = "back";
+  back.dataset.numberOfElement = i;
+  flipper.appendChild(back);
+}
+
+let backs = document.querySelectorAll(".back");
+
+generateBackGrn(backs, valuesOfBackPos);
+
+function generateBackGrn(elements, arrayOfBackGrnds) {
+  let arrayOfElements = Array.from(elements);
+  arrayOfElements.sort(function() {
+    return 0.5 - Math.random();
+  });
+  arrayOfElements.forEach((element, index) => {
+    if (index >= arrayOfBackGrnds.length) {
+      element.style.backgroundPosition =
```

Comment:
Don't use inline styles 
I give you a tip you can add class for this element)

Reply:
Refactored, but can you please explain, why is it bad ? In case of readabilty i guess ?

Reply:
Yes sure, [inline styles is bad](http://www.lostsaloon.com/technology/why-is-inline-css-bad-is-it-really-that-bad/)

Reply:
Yep,that makes sence.Good link ; )